ext/standard/tests/strings/str_decrement_basic.phpt

```diff
@@ -44,6 +53,22 @@ string(2) "Y9"
 string(2) "Z0"
 string(2) "y9"
 string(2) "z0"
+string(1) "z"
+string(2) "Aa"
+string(1) "Z"
+string(2) "aA"
+string(1) "9"
+string(2) "A0"
+string(1) "9"
+string(2) "a0"
+string(1) "9"
+string(2) "10"
+string(1) "Z"
+string(2) "1A"
+string(1) "z"
+string(2) "1a"
+string(2) "9z"
+string(3) "10a"
```

Comment:
This seems to have no effect ``"10a"--`` resulting in ``"10a"``

Reply:
Oh, I'll have a look later.

Reply:
Oh, this already works, the right value is right above it ^^

Reply:
Ah right I misread the test